docker-compose.yaml

```diff
@@ -4,6 +4,9 @@ services:
     build:
       context: .
       dockerfile: Dockerfile
+    depends_on: 
+      - gateway-inbound
```

Comment:
Task: I'm a big fan on the `depends_on` addition, but I think the `gateway-inbound` container actually depends on the `app` container.

Reply:
Question: What's the purpose of adding this? Are things not starting up correctly?

Reply:
Things started fine, but this is just more precise and safe for consumers of this project.